planning/behavior_path_lane_change_module/src/interface.cpp

```diff
@@ -122,16 +122,19 @@ BehaviorModuleOutput LaneChangeInterface::plan()
 
 BehaviorModuleOutput LaneChangeInterface::planWaitingApproval()
 {
+  module_type_->updateLaneChangeStatus();
```

Comment:
`updateLaneChangeStatus()` is called in `updateData()` so I think this can be removed.

Reply:
Thanks. Done.
https://github.com/autowarefoundation/autoware.universe/pull/6176/commits/f2b605f2260ca3a047d3917740719b21a5ece04d